2d_classification/monai_101.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.            ToTensorD(keys=["image", "label"]),
</pre><p>maybe there's no need to recommend using <code>ToTensorD</code> any more? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Oh, yes.
@mingxin-zheng Please delete this line some other tutorial PRs.

Thanks in advance.

Reply:
Fixed the commented line. Thanks!